README.md

```diff
@@ -20,6 +20,15 @@ __Testing:__
 * Tests written in [Jest](https://jestjs.io) and [Enzyme](https://enzymejs.github.io/enzyme/)
 * CI/CD done with [Github Actions](https://github.com/features/actions)
 
+## Project Setup
+Follow this quick start guide to get up and running. This guide assumes a working knowledge of npm and git. Make sure to check out the __CONTRIBUTING.md__ as well.
+* Clone the example-client repository: `git clone git@github.com:LibertyDSNP/example-client.git`
+* Install asdf: `asdf install`
```

Comment:
asdf is not a requirement, but just a common way to install the correct node version. Not sure how we want to communicate that however.